service/history/timer_queue_standby_task_executor.go

```diff
@@ -444,7 +474,6 @@ func (t *timerQueueStandbyTaskExecutor) executeWorkflowExecutionTimeoutTask(
 		wfContext workflow.Context,
 		mutableState workflow.MutableState,
 	) (interface{}, error) {
-
 		if !t.isValidExecutionTimeoutTask(mutableState, timerTask) {
```

Comment:
`t.isValidExecutionTimeoutTask` already checks if the workflow is running, so no need to add the check here again.